roles/openshift_node_group/templates/node-config.yaml.j2

```diff
@@ -35,19 +46,12 @@ masterKubeConfig: node.kubeconfig
 networkConfig:
   mtu: {{ openshift_node_group_network_mtu }}
   networkPluginName: {{ openshift_node_group_network_plugin }}
-nodeIP: ""
-podManifestConfig: null
+networkPluginName: {{ openshift_node_group_network_plugin }}
```

Comment:
@smarterclayton isn't this new networkPluginName a dupe?  it's specified just above in networkConfig already.

Reply:
What requirements does kubenet have?